src/test/java/rest/gorestcoIn/utils/helpermethods/APICal.java

```diff
@@ -0,0 +1,35 @@
+package rest.gorestcoIn.utils.helpermethods;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+
+public class APICal {
+
+    public static Response post(String jsonBody, Header header) {
+
+        return RestAssured
+                .given()
+                .header(header)
+                .contentType(ContentType.JSON)
+                .body(jsonBody)
+                .post("users")
+                .thenReturn();
+    }
+
+    public static Response get(int id) {
```

Comment:
this and the next methods also should have wider usage, bypass hardcoding as much as possible